tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -684,23 +705,23 @@ public static function provideFix80Cases(): iterable
         ];
 
         yield [
-            '<?php function foo(A|B|C $x) {}',
             '<?php function foo(\A|\B|\C $x) {}',
+            '<?php function foo(A|B|C $x) {}',
```

Comment:
I said that leading backslash behavior must be done as opt-in, but new option is implemented contrariwise so default value causes `\` being added, which is wrong. All changes in test cases must be reverted to previous state, only new cases must be added with non-default value to show new, optional behavior.